pkg/messages/takeFoodMessage.go

```diff
@@ -0,0 +1,19 @@
+package messages
+
+type takeFoodMessage struct{
+	*baseMessage
+	Food int
```

Comment:
```suggestion
	Food int
	Reason string
```

Could be useful for when treaties need to be broken (due to desperation/inevitable death) to prevent a loss in trust/opinion. Would need others' opinions